packages/browser-wallet/src/background/confirmation.ts

```diff
@@ -80,7 +67,20 @@ async function monitorCredentialStatus(initialNetwork: NetworkConfiguration, cre
                 genesisHash
             );
             return false;
-        } catch {
+        } catch (e) {
+            // transaction has been discarded by the node.
+            if ((e as RpcError).code === 'NOT_FOUND') {
```

Comment:
It would be preferable if the `NOT_FOUND` was gotten directly from an enum in the library, but as far as I can see it is not available - they just link to the grpc repository file with the enum.